solutions/frontend/template.yaml

```diff
@@ -64,11 +84,22 @@ Conditions:
       - Fn::Equals:
           - !Ref PermissionsBoundary
           - "none"
+  CanariesEnabled: !Equals
+    - !FindInMap [EnvironmentConfiguration, !Ref Environment, UseCanary]
+    - true
 
 Globals:
   Function:
-    CodeUri: dist
-    Handler: lambda.handler
+    DeploymentPreference:
```

Comment:
Eventually we will need to define DeploymentPreference per lambda in order to define the alarms that trigger a rollback